mojap_metadata/metadata/specs/table_schema.json

```diff
@@ -305,18 +305,25 @@
             ]
         },
         "struct_types": {
-            "type": "object",
-            "properties": {
-                "type": {
-                    "type": "string",
-                    "pattern": "^map_<.+>$|^struct<.+>$"
+            "allOf": [
+                {
+                    "$ref": "#/definitions/additional_col_properties"
```

Comment:
This was missing from the schema but is meant to be there like all the other column types.